demo/package.json

```diff
@@ -6,32 +6,34 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "web": "expo start --web",
-    "eject": "expo eject"
+    "web": "expo start --web"
   },
   "dependencies": {
-    "@react-native-community/datetimepicker": "6.1.2",
-    "@react-native-picker/picker": "2.4.0",
-    "@react-navigation/bottom-tabs": "6.5.7",
-    "@react-navigation/native": "6.1.6",
-    "@react-navigation/stack": "6.3.16",
-    "expo": "~45.0.0",
-    "expo-status-bar": "~1.3.0",
-    "hyperview": "0.68.1",
+    "@babel/preset-env": "^7.22.9",
+    "@react-native-community/datetimepicker": "7.2.0",
+    "@react-native-picker/picker": "2.4.10",
+    "@react-navigation/bottom-tabs": "^6.5.8",
+    "@react-navigation/native": "^6.1.7",
+    "@react-navigation/stack": "^6.3.17",
+    "@types/react": "^18.2.18",
+    "expo": "~49.0.5",
+    "expo-status-bar": "~1.6.0",
+    "hyperview": "^0.68.1",
     "moment": "^2.29.4",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.68.2",
-    "react-native-gesture-handler": "2.11.0",
+    "react": "18.2.0",
+    "react-dom": "^18.2.0",
+    "react-native": "0.72.3",
+    "react-native-gesture-handler": "~2.12.0",
```

Comment:
Similar to above, I wouldn't bump versions unless it's necessary for some feature